gemini/master.py

```diff
@@ -50,6 +50,24 @@ def join() -> str:
 
     return jsonify(response)
 
+@app.route('/v1/nodes/auth', methods=['GET', 'POST'])
+def auth() -> str:
+    node_id = request.args.get('node_id')
+    password = request.get_json('password')
+
+    cdata = c.kv.get('nodes/' + node_id)
+    cvalues = json.loads(cdata[1]['Value'])
+
+    if bcrypt.checkpw(password['password'].encode('utf8'), cvalues['password'].encode('utf8')):
```

Comment:
`password` is `request.get_json('password')`, and you're getting the value at index `password`, what this mean?